backend/src/main/resources/config/config.yml

```diff
@@ -9,6 +9,8 @@ map:
 geocoding:
   enabled: false
   host: nominatim.maps.tuerantuer.org
+csvWriter:
```

Comment:
Minor: Add a comment what this is doing, e.g. where the csvs are written to, and when to use the csvWriter.